docs/source/running-jobs/distributed-jobs.rst

```diff
@@ -54,3 +60,27 @@ SkyPilot exposes these environment variables that can be accessed in a task's ``
     :code:`run` command with :code:`echo $SKY_NODE_IPS >> ~/sky_node_ips`.
 - :code:`SKY_NUM_GPUS_PER_NODE`: number of GPUs reserved on each node to execute the
   task; the same as the count in ``accelerators: <name>:<count>`` (rounded up if a fraction).
+
+
+Executing a multi-node task
+-----------------------------------------
+
+The execution behavior is: all nodes are provisioned (barrier);
+workdir/file_mounts are synced to all nodes (barrier); ``setup`` commands are
+executed on all nodes (barrier); finally, the ``run`` commands are executed on
```

Comment:
This was a little difficult to read. Consider using bullet points:
```
The execution sequence on submitting a multi-node task is:

1. Nodes are provisioned. SkyPilot waits for all nodes to be ready.
2. Workdir/file_mounts are synced to all nodes. SkyPilot waits for completion of all file transfers.
3. Setup commands are executed on all nodes. SkyPilot waits for setup to complete on all nodes.
4. Run commands are executed on all nodes.
```

Reply:
I put in bullet points and tried two sentences in each. It read too long, so opted for "(barrier)".